### DISCUSSION TITLE
How to show space between two words of MetroWindow Title?

### DISCUSSION THREAD

**Discussion by omeraziz** [MahApps/MahApps.Metro#4135] [Q&A]
I have to show title of MetroWindow as "Application Pro". MahApps automatically deletes space and it becomes "ApplicationPro". How to fix this?

**Comment by punker76** [comment]
@omeraziz Can you post the Xaml or code behind of your main window? Normally the spaces are shown like in the main demo:

![image](https://user-images.githubusercontent.com/658431/123073552-cc7c4f80-d416-11eb-8f65-3383a97cce1f.png)

**Comment by timunie** [comment]
I think if you have more than one space WPF will remove any additional space. This is not related to MahApps. Plese find more info and how to deal with it here https://docs.microsoft.com/en-us/dotnet/desktop/xaml-services/white-space-processing

Happy coding
Tim
